README.rst

```diff
@@ -1603,6 +1956,13 @@ The ``AlertSettings`` of ``ping`` metric will by default use ``threshold`` and `
 defined in the ``alert_settings`` key.
 You can always override them and define your own custom values via the *admin*.
 
+You can also use ``alert_on_related_field`` key in metric configuration
+which allows ``AlertSettings`` to use ``related_field`` as value to check ``threshold`` instead
+of default ``field_name`` key. A real world example of this can be seen in 
+`Iperf metric configuration 
+<https://github.com/openwisp/openwisp-monitoring/blob/issue-385/iperf-check/openwisp_monitoring/monitoring/configuration.py#L546-L700>`_,
```

Comment:
definitely we cannot link files which are published in branches we are going to delete, please double check the rest of the document for similar problems, we can point to master here, even thought it won't work until published, but that's ok.